docs/didkit/did-web.md

```diff
@@ -0,0 +1,153 @@
+---
+id: did-web
+title: Did-web in minutes
```

Comment:
Is that the correct way to capitalise DID-web did:web Did-Web did-web?

Reply:
good catch